topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.md

```diff
@@ -0,0 +1,368 @@
+> 1. What is the quality of your sequences?
+> 2. Do you have adapters?
+{: .question}
+
+# Cleaning the Data
+
+## Cutadapt
+Cutadapt enables the removal of adapters, polyA tails, and other artifacts from sequences. The tool also filters reads based on quality.
+
+> <hands-on-title> Cutadapt </hands-on-title>
+>
+> 1. {% tool [Cutadapt](toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt/4.8+galaxy0) %} with the following parameters:
+>
+>     - **"Single-end or Paired-end reads?"**: `Single-end`
+>     - In **"Read Modification Options"**:
+>         - **"Quality cutoff"**: `30`
+>         - **"Shortening reads to a fixed length"**: `Disabled`
+>
+> > <comment-title> Suggestions </comment-title>
+> >
+> > You may consider changing these parameters depending on the quality of your dataset.
+> {: .comment}
+>
+{: .hands_on}
+
+>    > <comment-title> Quality Control </comment-title>
+>    >
+>    > We do a second quality control similar to the first one to check the quality of the sequences after cleaning them.
+>    {: .comment}
+
+## Quality Control with FASTQC and MultiQC
+
+> <hands-on-title> FastQC </hands-on-title>
+>
+> 1. {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy0) %} on the cutadapt output files>
+>
+>
+> 2. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.11+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>        - {% icon param-repeat %} *"Insert Results"*
+>            - *"Which tool was used generate logs?"*: `FastQC`
+>
+>
+>    > <comment-title> Comment </comment-title>
+>    >
+>    > You should notice an improvement on the quality of your sequences.
+>    {: .comment}
+>
+{: .hands_on}
+
+## Filtering the collection
+
+> <hands-on-title> Filter empty datasets </hands-on-title>
+>
+> 1. {% tool [Filter empty datasets](__FILTER_EMPTY_DATASETS__) %}
+>
+>
+> 2. {% tool [Filter FASTQ](toolshed.g2.bx.psu.edu/repos/devteam/fastq_filter/fastq_filter/1.1.5) %} with the following parameters:
+>    - *"Minimum size"*: `300`
+>
+>    > <comment-title> Comment </comment-title>
+>    >
+>    > Here we descide to only keep sequences of 300bp or above, you may change this parameter depending on your dataset
+>    {: .comment}
+>
+{: .hands_on}
+
+### Changing files names
+
+> <hands-on-title>Extract element identifiers </hands-on-title>
+>
+> 1. {% tool [Extract element identifiers](toolshed.g2.bx.psu.edu/repos/iuc/collection_element_identifiers/collection_element_identifiers/0.0.2) %}
+>
+> 2. {% tool [Regex Find And Replace](toolshed.g2.bx.psu.edu/repos/galaxyp/regex_find_replace/regex1/1.0.3) %} with the following parameters:
+>    - In *"Check"*:
+>      - {% icon param-repeat %} *"Insert Check"*
+>        - *"Find Regex"*: `.ab1`
+>        - *"Replacement"*: `.fastq.gz`
+>
+>    > <comment-title> Comment </comment-title>
+>    >
+>    > This is to ensure that all your files names end with .fastq.gz
+>    {: .comment}
+>
+>
+> 3. {% tool [Paste](Paste1) %}

```

Comment:
parameters for this tool? which files need to be pasted and in which order?

Reply:
Done! Thank you!